README.md

```diff
@@ -0,0 +1 @@
+# WHAT IS THIS?

```

Comment:
It's a read me file. Ha

Its an extension for blt commands. I attempted to add them to caravan but it wasn't working. I believe there's a special namespacing convention required since it works with the other branch in this repo.

This way we can have a shared set of commands for all stacks instead of copying them.

Reply:
See https://github.com/SU-SWS/blt-sws/tree/blt-commmands